.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -0,0 +1,110 @@
+name: "Bug Report"
+description: Use this template to report a bug
+labels: ["bug"]
+body:
+  - type: markdown
+    id: example
+    attributes:
+      value: >
+        Thanks for reporting.
+        
+        - Make sure you are able to reproduce this issue on the latest released version of [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+        
+        - Please search the existing issues to see if there has been a similar issue filed.
+        
+        - If you want to discover useful tips on how to avoid common pitfalls and use the Azure CLI successfully, please refer to [Tips for using the Azure CLI successfully](https://learn.microsoft.com/en-us/cli/azure/use-cli-effectively).
+  - type: textarea
+    id: desc
+    attributes:
+      label: Describe the bug
+      description: >
+        Please describe the bug.
+      placeholder: >
+        A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: cmd
+    attributes:
+      label: Related command
+      description: Please provide the related commands and parameters with az {command} if you can, so that we can quickly route to the related team for help.
+      placeholder: e.g., `az vm create --image`
+    validations:
+      required: true
+  - type: textarea
+    id: error
+    attributes:
+      label: Errors
+      description: >
+        Please paste here the error message you have received. 
+        
+        **⚠ ATTENTION:** Make sure to remove all sensitive information, such as username, password, credential, subscription ID, etc.
+      placeholder: >
+        The command failed with an unexpected error. 
+        Here is the traceback: {The rest of the error message}
+    validations:
+      required: true
+  - type: textarea
+    id: output
+    attributes:
+      label: Issue script & Debug output
+      description: >
+        Append `--debug` to each command in the script and rerun the script and paste the debug log here
+        
+        **⚠ ATTENTION:** Be sure to remove any sensitive information that may be in the logs
+      placeholder: >
```

Comment:
What's the difference between `placeholder: >` and `placeholder: |` ? I saw Az PS uses `placeholder: |`. https://github.com/Azure/azure-powershell/blob/main/.github/ISSUE_TEMPLATE/1-AZ-BUG-REPORT.yml?plain=1

Reply:
Values can span multiple lines using `|` or `>`. 
Spanning multiple lines using a “Literal Block Scalar” `|` will include the newlines and any trailing spaces. 
Using a “Folded Block Scalar” `>` will fold newlines to spaces; it’s used to make what would otherwise be a very long line easier to read and edit.
The left half of the figure below is all using `>`, and the right half is all using `|`.
![image](https://github.com/Azure/azure-cli/assets/18628534/39737689-1185-436c-a388-e52e1dbcf002)